src/test/test_verify.hpp

```
@@ -113,8 +110,57 @@ inline std::vector<prevail::RawProgram> read_elf_cached(const std::string& path,
+            for (size_t i = 0; i < N; ++i) {
+                data[i] = str[i];
+            }
+        } else {
+            // FNV-1a hash of full name for a unique suffix.
+            uint32_t hash = 2166136261u;
+            for (size_t i = 0; i < N - 1; ++i) {
+                hash ^= static_cast<unsigned char>(str[i]);
+                hash *= 16777619u;
+            }
+            // Format: first (MAX_LEN - 6) chars + "..xxxx"
+            constexpr size_t prefix_len = MAX_LEN - 6;
```

Clang (14.0.6)

[warning] variable 'prefix_len' is not initialized

(cppcoreguidelines-init-variables)